.github/workflows/puppetfile.yml

```diff
@@ -0,0 +1,73 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
I think a lot of these comments are redundant and would prefer to drop them.

Reply:
I will drop them on next push.